sw/CMakeLists.txt

```diff
@@ -42,10 +42,10 @@ if(NOT WIN32)
 endif()
 
 # set the project name
-project(${PROJECT} ASM C)
+project(${PROJECT} ASM C CXX)
 
 # set the required CMake standard
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
can't we keep the same CMake version?

Reply:
It should also work with 2014

Reply:
If that is the case, can you revert this change pls ? Also, if you want to disclose it (you don't have to) can we know what is the project you are working on x-heep?

Reply:
Fixed. We are working on our own AI accelerators for IoT (ultra-low-power), it's part of the A-IQ Ready project. We're in tight collaboration with UPM

Reply:
the name of the company is NVISION